docs/commands/count.md

```diff
@@ -32,14 +36,18 @@ This command counts the number of rows in a table.
 ────┴────────────────────┴──────┴──────────┴──────────────
 ```
 
+By default, `count` will return the number of rows in a table
+
 ```shell
 > ls | count
 20
 ```
 
+The `-c` flag will produce a count of the columns in the table
+
 ```shell
-> ls | get name | count
```

Comment:
I just deleted this example in the documentation because it seems to be pretty much the same thing as the `ls | count` example